tools/llvm-slicer-opts.cpp

```diff
@@ -87,6 +87,10 @@ SlicerOptions parseSlicerOptions(int argc, char *argv[]) {
         llvm::cl::desc("Perform forward slicing\n"),
                        llvm::cl::init(false), llvm::cl::cat(SlicingOpts));
 
+    llvm::cl::opt<bool> threads("threads",
+        llvm::cl::desc("Consider threads are in input file (default=false)."),
```

Comment:
The message seems a bit ambiguous... what about somethink like: "Enable support for threaded programs"? or  "Enable support for analysis of programs using pthread API"?